app/plugin.go

```diff
@@ -666,6 +737,10 @@ func (a *App) getPluginsFromFolder() (map[string]*pluginSignaturePaths, *model.A
 	if appErr != nil {
 		return nil, model.NewAppError("getPluginsFromDir", "app.plugin.sync.list_filestore.app_error", nil, appErr.Error(), http.StatusInternalServerError)
 	}
+	return getPluginsFromFilePaths(fileStorePaths), nil
+}
+
+func getPluginsFromFilePaths(fileStorePaths []string) map[string]*pluginSignaturePaths {
```

Comment:
I would like us to not pass pointer all the time. What do you think about just using a ` map[string]pluginSignaturePaths`?

Reply:
I like passing pointer better, because you will be copying just 8 bytes not the whole structs. Reference magic is done by a compiler anyway so I don't see performance problems there.
As I understand we use pointers in other places as well.

Reply:
I know the arguments listed, in this case I'm still in favor of using pointers. We can discuss whole pointer issue in the next sprint planning. I don't think it's a blocking concern for this PR(We can come back to it when we merge to master).

Reply:
:+1: that this is not a blocker.